deployment/pcf/config/config.js

```diff
@@ -6,7 +6,7 @@ window.Retro = {
     "stylesheets": ["application.css"],
     "useRevManifest": true,
     "api_base_url": "https://{{api-app-name}}.{{pcf-url}}",
-    "websocket_url": "wss://{{api-app-name}}.{{pcf-url}}:443/cable",
+    "websocket_url": "wss://{{api-app-name}}.{{pcf-url}}:4443/cable",
```

Comment:
This actually isn't a mistake. PWS runs (secure) websockets on 4443 but most PCF installations will run it on the standard 443. Could you revert this please?

Reply:
Ah, good catch, thanks. I deployed it to PWS, couldn't get my hands on a regular PCF. Reverted.